corehq/form_processor/migrations/0078_blobmeta_migrated_check.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2019-01-05 00:21
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from __future__ import print_function
+
+import sys
+import traceback
+
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq.sql_db.operations import HqRunPython, noop_migration_fn
+
+
+BLOBMETAS_NOT_MIGRATED_ERROR = """
+Blob metadata needs to be migrated before this environment can be upgraded to
+the latest version of CommCareHQ. Instructions for running the migration can be
+found at this link:
+
+https://github.com/dimagi/commcare-cloud/blob/master/docs/changelog/0009-blob-metadata-part-2.md
+
+If you are unable to run the management command because it has been deleted,
+you will need to checkout an older version of CommCareHQ first:
+
+git checkout acc40116a96a40c64efb8613fb2ba5933122b151
+"""
+
+
+def get_num_attachments(connection):
+    """Get the number of attachments that need to be migrated"""
+    with connection.cursor() as cursor:
+        cursor.execute("SELECT COUNT(*) FROM form_processor_xformattachmentsql")
```

Comment:
just for my own education, this just automatically gets called once for every shard during migrate?

Reply:
correct

Reply:
every migration is run on every database in `settings.DATABASES`